src/test/java/core/basesyntax/servicesimpl/ParseFruitNamesImplTest.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.servicesimpl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+import core.basesyntax.db.DataBase;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.services.ParseFruitNames;
+import java.util.HashMap;
+import java.util.List;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ParseFruitNamesImplTest {
+    private final ParseFruitNames parser = new ParseFruitNamesImpl();

```

Comment:
initialize such fields in @BeforeClass, fix it in every test class

Reply:
✅